index.bs

```diff
@@ -805,6 +805,57 @@ and doing cleanup as necessary (see above).
   is still [under discussion](https://github.com/whatwg/html/issues/6588).
 </div>
 
+<h3 id="third-party-tools">Handling conflicts with third-party tools</h3>
+
+The web platform benefits from a wide ecosystem of libraries, polyfills, preprocessors,
+and other tools to facilitate easier and faster development.
+A lot of the time, the syntax of an upcoming web platform feature
+may conflict with that of a third-party tool causing breakage.
+This is especially common as third-party tools are often used to prototype
+new web platform features.
+
+In some cases, the conflict will break the web too much,
+and the feature's syntax needs to be redesigned to avoid clashes.
```

Comment:
```suggestion
In some cases, the conflict will introduce problems across a large number of
web sites, necessitating the feature's syntax to be redesigned to avoid clashes.
```